artifactory_test.go

```diff
@@ -62,6 +63,9 @@ import (
 // JFrog CLI for Artifactory commands
 var artifactoryCli *tests.JfrogCli
 
+// JFrog CLI for Artifactory commands
```

Comment:
This description should be changed, because it is the exact same description of the above ```artifactoryCli``` variable. Let'e add a description that makes it easier to understand the difference between the two variables.